lib/inferno/dsl/fhir_evaluation/rules/all_must_supports_present.rb

```diff
@@ -46,9 +46,9 @@ def check(context)
             if missing_items_by_profile.count.zero?
               result = EvaluationResult.new('All MustSupports are present', severity: 'success', rule: self)
             else
-              message = 'Found Profiles with not all MustSupports represented:'
+              message = 'Found Profiles with not all MustSupports presented:'
```

Comment:
```suggestion
              message = 'Found Profiles with not all MustSupports represented:'
```
I think this one should be reverted.